NewsFeed/services/CCBRNewsDataStore.m

```diff
@@ -42,11 +42,17 @@ - (void)loadNextArticles {
         self.loading = NO;
         
         if (error) {
-            // TODO: Handle error
+            if (self.nextArticlesCallback) {
+                self.errorCallBack(error.description);
```

Comment:
Because we're calling `errorCallBack`, we should check the validation of `errorCallBack` instead of `nextArticlesCallback`.